clang/lib/Sema/SemaAPINotes.cpp

```diff
@@ -511,6 +511,10 @@ static void ProcessAPINotes(Sema &S, FunctionOrMethod AnyFunc,
       AnyTypeChanged = true;
   }
 
+  // returns_(un)retained
+  if (!Info.SwiftReturnOwnership.empty())
+    D->addAttr(SwiftAttrAttr::Create(S.Context, Info.SwiftReturnOwnership));
```

Comment:
I think we would want to construct a string `"returns_" + Info.SwiftReturnOwnership` here.